### ISSUE TITLE
Ability to retrieve the original value of an overridden preference

### ISSUE THREAD

**Issue by kizu** [WICG/web-preferences-api#38]
(Copying from a mastodon thread with some rewording — https://front-end.social/@kizu/113160248251763608)

There are legit cases when you'd want to override the value that is used everywhere, but then still be able to differentiate how things look/behave based both on the original value, and on the override. See https://blog.kizu.dev/querying-the-color-scheme/#not-the-user-preference, for example — it is very similar to what we'll have with the web preferences.

With the way things are specified right now, we could do it in a hacky way: clear the override, read the value, restore the override, but that feels very cumbersome and inconvenient.

It would be great if we could get the original value natively (and also monitor it similar to `matchMedia` maybe).

**Comment by bramus** [comment]
While I understand the use-case, this — unlike the override itself — would add an extra bit of tracking data.

By only having access to the overridden value, entropy is added to the tracking data. If we were to expose the original value, the opposite happens.

**Comment by lukewarlow** [comment]
I don't think this does actually introduce new entropy? Exactly because the author can just do what's described, clear the value, read it and then override again? I'm going to close this issue and move it to the CSSWG for further discussion.

See https://github.com/w3c/csswg-drafts/issues/11580
